src/main/java/com/sysops/dao/ArticleDao.java

```diff
@@ -0,0 +1,12 @@
+package com.sysops.dao;
+
+import com.sysops.entity.Article;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface ArticleDao extends CrudRepository<Article, Long> {
+    @Query("SELECT article FROM Article a WHERE a.title LIKE %:keyword% OR a.content LIKE %:keyword%")
+    List<Article> findByKeyword(String keyword);
```

Comment:
I forgot to add "@Param("keyword")" to the parameter. Will include it in later commit.